include/config/battle.h

```diff
@@ -182,6 +182,8 @@
 // Eg: Replace with VAR_UNUSED_0x40F7 so you can use B_VAR_TERRAIN for that feature.
 #define B_VAR_TERRAIN               0     // If this var has a value, assigning a STATUS_FIELD_xx_TERRAIN to it before battle causes the battle to start with that terrain active.
 #define B_VAR_TERRAIN_TIMER         0     // If this var has a value greater or equal than 1 field terrains will last that number of turns, otherwise they will last until they're overwritten.
+#define B_SET_TRICK_ROOM            0     // If this flag is set, Trick Room will start before the battle.
```

Comment:
is there a use case for having trick room be different from either 5 turns or infinite? I would think two flags would suffice. Or just a single variable of the turn counter, where 0xFF is infinite. A flag and var seems unecessary

Reply:
You mean one flag? I don't disagree

Reply:
I pushed a change. Hope I understood the comment correctly.